sw/device/tests/aon_timer_irq_test.c

```diff
@@ -92,11 +92,13 @@ static void execute_test(dif_aon_timer_t *aon_timer, uint64_t irq_time_us,
   // The interrupt time should be `irq_time_us ±5%`.
   uint64_t variation = irq_time_us * 5 / 100;
   CHECK(variation > 0);
-  // Add 200us of overhead on the irq handling
-  variation += 200;
   uint64_t sleep_range_h = irq_time_us + variation;
   uint64_t sleep_range_l = irq_time_us - variation;
 
+  // Add 500 cpu cycles of overhead to cover irq handling.
+  sleep_range_h += 500 * 1000000 / kClockFreqCpuHz;
+  sleep_range_l += 500 * 1000000 / kClockFreqCpuHz;
```

Comment:
Shouldn't the lower bound be zero? Otherwise, you're asserting that IRQ handling takes exactly 500 cycles...